### PR TITLE
update the release process docs for interpret-community

### PR DESCRIPTION
**Pull request by imatiach-msft**
Some minor updates to the release process docs

### CODE REVIEW

File: docs/release-process.md [hunk 1]
```diff
@@ -28,6 +28,18 @@ For a guide on the pypi release process, please see:
 
 https://packaging.python.org/tutorials/packaging-projects/
 
+## Pipeline automatic release process
+
+Go to the PYPI-Release pipeline located at the build definition:
+
+https://dev.azure.com/responsibleai/interpret-community/_build?definitionId=70
+
+Sign in and trigger a release by selecting the "Run pipeline" button.  Select Release Type = "Test" to trigger a test release.  After build is complete and validated, select Release Type = "Production" to release to pypi prod.
```
Comment (romanlutz) [inline docs/release-process.md:37]:
```suggestion
Sign in and trigger a release by selecting the "Run pipeline" button.  Select Release Type = "Test" to trigger a test release.  After build is complete and validated, select Release Type = "Production" to release to PyPI prod.
```

File: docs/release-process.md [hunk 2]
```diff
@@ -28,6 +28,18 @@ For a guide on the pypi release process, please see:
 
 https://packaging.python.org/tutorials/packaging-projects/
 
+## Pipeline automatic release process
+
+Go to the PYPI-Release pipeline located at the build definition:
```
Comment (romanlutz) [inline docs/release-process.md:33]:
```suggestion
Go to the PyPI-Release pipeline located at the build definition:
```

File: docs/release-process.md [hunk 3]
```diff
@@ -28,6 +28,18 @@ For a guide on the pypi release process, please see:
...
+
+https://dev.azure.com/responsibleai/interpret-community/_build?definitionId=70
+
+Sign in and trigger a release by selecting the "Run pipeline" button.  Select Release Type = "Test" to trigger a test release.  After build is complete and validated, select Release Type = "Production" to release to pypi prod.
+
+## Manual process
+
+If the release pipeline is not working or there is some justification to release manually, the old direct release process can be used with local commands outlined below.
+
 ### pypi file
```
Comment (romanlutz) [inline docs/release-process.md:43]:
```suggestion
### PyPI file
```

File: docs/release-process.md [hunk 4]
```diff
@@ -85,6 +97,11 @@ dist/
 
 Upgrade twine before uploading to pypi:
```
Comment (romanlutz) [inline docs/release-process.md:98]:
```suggestion
Upgrade twine before uploading to PyPI:
```

**Review by romanlutz — APPROVED** [review]
Thanks for updating! I put in some suggestions for fixing the casing of PyPI which you can add or ignore 🙂

File: docs/release-process.md (hunk 2 above)
Comment (imatiach-msft) [reply docs/release-process.md:33]: done!

File: docs/release-process.md (hunk 1 above)
Comment (imatiach-msft) [reply docs/release-process.md:37]: done!

File: docs/release-process.md (hunk 3 above)
Comment (imatiach-msft) [reply docs/release-process.md:43]: done!

File: docs/release-process.md (hunk 4 above)
Comment (imatiach-msft) [reply docs/release-process.md:98]: done!

**Pull request merged**